packages/dds/tree/src/simple-tree/core/context.ts

```diff
@@ -15,11 +15,41 @@ import { walkAllowedTypes } from "./walkSchema.js";
 /**
  * Creating multiple simple tree contexts for the same branch, and thus with the same underlying AnchorSet does not work due to how TreeNode caching works.
  * This slot is used to detect if one already exists and error if creating a second.
+ * @remarks
+ * See also {@link ContextSlot} in which the flex-tree context is stored.
  */
 export const SimpleContextSlot = anchorSlot<HydratedContext>();
 
+/**
+ * Additional information about a collection of {@link TreeNode}s.
+ * @remarks
+ * Each TreeNode provides a way to navigate to a context for it with additional information.
+ *
+ * For unhydrated nodes, this information is rather limited since the node doesn't know what tree it might get inserted into (if any),
+ * and thus is limited to information derived about that particular unhydrated tree and its schema.
+ *
+ * Hydrated nodes have more contextual information, and thus can provide a single {@link HydratedContext} for all nodes in the document which can have additional information.
+ *
+ * @privateRemarks
+ * This design is the same as {@link FlexTreeContext} with its base type and {@link FlexTreeHydratedContext} extending it.
+ */
 export class Context {
+	/**
+	 * All schema which could transitively be used under the node who's context this is.
```

Comment:
Nit (maybe a bit simpler)
```suggestion
	 * All schema which could transitively be used under the associated node.
```